src/main/java/CalApp.java

```diff
@@ -0,0 +1,19 @@
+import java.util.Scanner;
+
+import calculator.Calculator;
+
+public class CalApp {
+	public static void main(String[] args) {
+
+		/* input value */
+		Scanner sc = new Scanner(System.in);
+		String calStr = sc.nextLine();
```

Comment:
변수명은 축약하지 않는 편이 좋습니다. (물론 실무에서는 과도하게 길다면 축약하여 사용할 수 있겠지만..)
그리고 `String` 이라는 타입이므로 Str이라는 네이밍이 중복됩니다.
```suggestion
		String input = sc.nextLine();
```